README.md

```diff
@@ -1,8 +1,115 @@
+
 # supported ![CI](https://github.com/stefanpenner/supported/workflows/CI/badge.svg)
 
 
+
+
 ## Usage
 
+
 ```sh
 npx supported <path/to/node_module>
+npx supported <[array/of/node_modules]>
+```
+
+### As a node module
+
+
+```js
+const { isAllInSupportWindow } = require('supported');
```

Comment:
can we nit-pick this API name?

* `is` to be implies, true/false return value, but we get a complex structure back
* `all` all of what?

The way I think about a function name:
A function name is the relationship between the functions inputs, and the functions (outputs, or side-effects).

Given:

```js
let b = isAllInSupportWindow(a);
```

I read this as  `B` will represent true or false based on if all of `A` is in the support window. Rather, it appears that the relationship between `A` and `B` is more `B` is the result of running support policies across `A`. And then B can be inspected for the result of the policies.

Given this, any ideas on an improved function name?

Reply:
Can we rename it to `getSupportPolicyResult`?